backend/trading/features/calculators.py

```
@@ -0,0 +1,585 @@
+        
+        vol = np.sqrt(variance)
+        
+        if annualize:
+            vol *= np.sqrt(252)
+        
+        return vol
+    
+    @staticmethod
+    def momentum(
+        prices: np.ndarray,
+        lookbacks: list = [5, 10, 20, 60]
```

Ruff (0.14.8)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)